database/migrations/2020_05_13_134740_create_members_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMembersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('members', function (Blueprint $table) {
```

Comment:
ơ cái này lưu luôn cả thông tin đăng nhập vào bảng members à e ? chứ ko tách riêng thông tin đăng nhập riêng ở bảng users còn mỗi user sẽ có 1 member profile để tham gia dự án à ? :D

Reply:
Vâng anh, em lưu hết toàn bộ vào member ạ